Resources/Storage/StorageFactory.py

```diff
@@ -89,7 +89,7 @@ def getStorages( self, storageName, pluginList = None ):
     if pluginList is None:
       pluginList = []
     if not self.vo:
-      return S_ERROR( 'Mandatory vo parameter is not defined' )
+      gLogger.warn( 'No VO information available' )
```

Comment:
Why is this still mandatory in v6r13? Changing this also allows us to download sandboxes as diracAdmin.